app/workers/stat/library_create_worker.rb

```diff
@@ -0,0 +1,25 @@
+class Stat
+  class LibraryCreateWorker
+    include Sidekiq::Worker
+
+    def perform(kind, user, library_entry_id)
+      library_entry = LibraryEntry.find(library_entry_id)
+      library_event = LibraryEvent.create_for(:added, library_entry)
```

Comment:
uhhhhh I don't think this'll work

iirc `LibraryEvent.create_for` uses the dirty tracking from ActiveRecord, but since this is in a background worker, it won't have the changes that the frontend made